### DISCUSSION TITLE
Can't import externally downloaded Mini_ finality_ stateroots_ latest. Car, what should I do

### DISCUSSION THREAD

**Discussion by coder-chenda** [filecoin-project/lotus#8562] [Build and Sync] [answered]
The situation is like this. Recently, the download of block photos imported by URL is very slow, so the complete minimal is downloaded directly on the server_ finality_ stateroots_ latest. car ，

Lotus daemon --import-snapshot   /absolutepath/minimal_finality_stateroots_latest.car   ，

Prompt when starting the node to prepare for synchronization that the minimum cannot be found  minimal_finality_stateroots_latest.car

ERROR: open minimal_finality_stateroots_1751760_2022-04-25_04-00-00.car: no such file or directory

![image](https://user-images.githubusercontent.com/83049018/165664538-056c1a53-09c7-4c02-ad34-e9b0f54e7952.png)

**Comment by stuberman** [comment]
Can you post the command you first used to start this node?
Do you know where you downloaded the car to?

**Reply by coder-chenda** [reply]
The first start command is:

lotus daemon --import-snapshot https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car



I download minimal_ finality_ stateroots_ latest. Car is in the current directory, and I use the absolute path to represent the import path of car

**Reply by stuberman** [reply]
My preference is to first download the new chain fully before using import:

1. Find name of newest chain: `curl -sI https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car | perl -ne '/x-amz-website-redirect-location:\s(.+)\.car/ && print "$1.car\n"'`

2. Download newest chain: `aria2c --max-connection-per-server=6 https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/<new chain name>`

When complete, then import chain: `lotus daemon --import-snapshot <newest chain> --halt-after-import `

**Reply by coder-chenda** [reply]
Please tell me specifically or take an example, for example, how to use the command to import and start the car package when downloading it to my server directory / home / Ubuntu / download /? Can you use the catalogue I provided to explain?

**Reply by coder-chenda** [reply]
<img width="943" alt="image" src="https://user-images.githubusercontent.com/83049018/166156441-44fa74c8-6fcb-4d48-b7ad-77b67821d28e.png">

**Reply by stuberman** [reply]
Where did you download it? use `ls -Alh` to check its location and name.
Then make sure you run the command from the same directory or include its full path.

**Reply by coder-chenda** [reply]
I use docker to start it. Do I need to import the downloaded car into the container?

**Reply by coder-chenda** [reply]
> Where did you download it? use `ls -Alh` to check its location and name. Then make sure you run the command from the same directory or include its full path.

I'd like to ask how to set the path of import snapshot to import data, because I already have data path L, and the default import will be / home / Ubuntu / Lotus / down, very upset

**Comment by LexLuthr** [comment, marked as answer]
@coder-chenda If are working with docker then you need to make the storage available in the container. How else will a process running inside the container be able to access the file.

**Comment by LexLuthr** [comment]
I am archiving this discussion as there has been no new input from the author for more than 7 days.
